firmware/mcal/cli/periph/pwm.h

```diff
@@ -30,15 +30,30 @@ class PWMOutput : public shared::periph::PWMOutput {
         std::cout << "Setting PWM " << name_ << " to " << duty_cycle_ << "%"
                   << std::endl;
     }
+
     float GetDutyCycle() override {
         std::cout << "PWM " << name_ << " has duty cycle " << duty_cycle_ << "%"
                   << std::endl;
         return duty_cycle_;
     }
 
+    void SetFrequency(float frequency) override {
+        frequency_ = frequency;
+
+        std::cout << "Setting frequency " << name_ << " to " << frequency_
```

Comment:
similar to above, can you make this print `Setting PWM "name_" frequency to __ Hz`